### PR TITLE
Override default configuration with in-repo config file

### PR DESCRIPTION
**Pull request by brbrr**
This PR adds a way to override default test-cases repository configuration, which will allow to reuse it in different projects

Testing Instructions:
- run unit tests :D 
- WIP

I also extracted helpers into a module, that should make testing a bit easier, but frankly, it is not needed for this PR. I'm happy to revert it back if desired.

### CODE REVIEW

**Comment by pinarol** [comment]
Changes look good to me but can you also update [README](https://github.com/wordpress-mobile/test-case-reminder-bot/blob/master/README.md) accordingly? 

- [ ] Do these changes effect anything already described there?

- [ ] What to do to override default test-cases repository configuration?

**Comment by brbrr** [comment]
Good catch! I completely forgot about updating README. I added the `Configuration` section in 0009253. Do you think it would be enough?

File: README.md [hunk 1]
```diff
@@ -1,40 +1,61 @@
...
+### Install
 
 To run the code, make sure you have [Bundler](http://gembundler.com/) installed; then enter `bundle install` on the command line.
 
-## Set environment variables
+### Set environment variables
 
 1. Create a copy of the `.env-example` file called `.env`.
-2. Add your GitHub App's private key, app ID, and webhook secret to the `.env` file.
+1. Create a test app with the following permissions: "Read access to code", "Read access to metadata", "Read and write access to pull requests"
```
Comment (pinarol) [inline README.md]:
```suggestion
2. Create a test app with the following permissions: "Read access to code", "Read access to metadata", "Read and write access to pull requests"
```

File: README.md [hunk 2]
```diff
@@ -1,40 +1,61 @@
...
+1. Add your GitHub App's private key, app ID, and webhook secret to the `.env` file.
```
Comment (pinarol) [inline README.md]:
```suggestion
3. Add your GitHub App's private key, app ID, and webhook secret to the `.env` file.
```

**Review by pinarol — APPROVED** [review]
Thank you for the docs, I just added suggestions to fix the bullet numbers. Feel free to :shipit: after applying them.

**Pull request merged**